pipelines/csw-test-pipeline.yml

```diff
@@ -40,7 +100,7 @@ resources:
     url: ((slack-webhook-cyber-test))
 
 jobs:
-- name: job-csw-test-build
+- name: build
```

Comment:
should this be called uat-build for consistency or is build simply the same for all env?

Reply:
this job is general, we could build any environment with this. But we are not going to use that for now.